docs/source/dataanalysis/dataanalysis.rst

```diff
@@ -59,6 +59,7 @@ Reduced Beam Characteristics
 ----------------------------
 
 ImpactX calculates reduced beam characteristics like averaged positions, momenta, beam emittances and Courant-Snyder (Twiss) parameters during runtime.
```

Comment:
Thanks Sarah, that is a good point.

Before adding the monitors we had some ad-hoc diagnostics (in ASCII) in ImpacX, where some of the legacy wording here comes from. We do not have a "full" diagnostics outside of monitors in ImpactX, for simplicity (still needs #537).

The "reduced diagnostics" (in situ diagnostics) is now also added to the monitors #584, so that we can technically streamline this even a bit more.

Let's add the cross-section hints you propose for now, to clarify what means what.

Reply:
I made an edit to line 61, so these comments now are invisible under the Files Changed tab.  Just FYI - these are good comments and we don't want to lose them.